numerics/poisson_series_basis_body.hpp

```diff
@@ -164,6 +174,20 @@ std::array<Series, sizeof...(indices)> AperiodicSeriesGenerator<
               {}))...};
 }
 
+template<typename Series, int degree, int dimension, std::size_t... indices>
+std::array<PoissonSeriesSubspace, sizeof...(indices)> AperiodicSeriesGenerator<
+    Series,
+    degree,
+    dimension,
+    std::index_sequence<indices...>>::Subspaces(Instant const& origin) {
+  return {PoissonSeriesSubspace{
+      static_cast<PoissonSeriesSubspace::Coordinate>(indices % dimension),
+      // The degree of the ith polynomial is i / dimension, so its parity is
```

Comment:
There is no `i`, it's called `indices`.

Reply:
Yes, but this sentence is illegible if expressed in terms of a pack.